portal-web/test/functional/com/liferay/portalweb/tests/enduser/segmentation/ExperienceCreateExperience.testcase

```diff
@@ -4556,9 +4556,15 @@ definition {
 	@ignore = "true"
 	@priority = "4"
 	test RequiredField {
+		task ("Create new experience") {
```

Comment:
In this task you are not "Creating a new experience", you are "Trying to create an experience with no name"